manager/apis/app/v1alpha1/m4dapplication_types.go

```diff
@@ -57,11 +58,17 @@ type ExternalDataContext struct {
 	// +optional
 	Recurrence RecurrenceType `json:"recurrence,omitempty"`
 }
+*/
 
 // DataContext indicates data set chosen by the Data Scientist to be read from or written to by his application,
 // and includes information about the data format and technologies used by the application
 // to access the data.
 type DataContext struct {
+	// DataCatalogID indicates the data catalog that contains the information about the data set.  If no value is provided the assumption is there is only one data catalog.
+	// TODO: Temporary pending design for multi-catalog support.
+	// +optional
+	DataCatalogID string `json:"dataCatalogID"`
```

Comment:
Can this be a part of DataSetID? Today DataSetID is a json that includes catalog id and asset id. We could add another attribute to identify the catalog (egeria, inline, ...)